app/angular/src/client/preview/angular-beta/RenderNgAppService.ts

```diff
@@ -63,9 +66,27 @@ export class RenderNgAppService {
    * - true render will only use the StoryFn `props' in storyProps observable that will update sotry's component/template properties. Improves performance without reloading the whole module&component if props changes
    * - false fully recharges or initializes angular module & component
    */
-  public async render(storyFn: StoryFn<StoryFnAngularReturnType>, forced: boolean) {
+  public async render(
+    storyFn: StoryFn<StoryFnAngularReturnType>,
+    { forced, parameters }: { forced: boolean; parameters: Parameters }
+  ) {
     const storyObj = storyFn();
 
+    if (storyObj.component) {
+      logger.warn(dedent`\`component\` story input is deprecated, and will be removed in Storybook 7.0.
+        Instead, use \`export const default = () => ({ component: AppComponent });\`
+        or
+        \`\`\`
+        export const Primary: Story = () => ({});
+        Primary.parameters = { component: AppComponent };
```

Comment:
don't hesitate if you have a better suggestion :) 

There are other possibilities :

```
storiesOf(...)
  .addParameters({
    component: AppComponent,
  })
```
or : 
```
.add(
    'background component',
    () => ({
      props: {},
    }),
    {
      component: AppComponent,
    }
  )
```
🤷‍♂️